src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/Logging/DocumentationSignatureGenerator.PartVisitor.cs

```diff
@@ -88,8 +88,10 @@ public void VisitMethodDefinition(MethodDefinition method, StringBuilder builder
                 if (method.HasGenericParameters)
                     builder.Append("``").Append(method.GenericParameters.Count);
 
-                if (method.HasParameters || (method.CallingConvention == MethodCallingConvention.VarArg))
+                if (method.HasMetadataParameters() || (method.CallingConvention == MethodCallingConvention.VarArg))
+#pragma warning disable RS0030 // MethodReference.Parameters is banned. This generates documentation signatures, so it's okay to use it here
```

Comment:
Do we need this?

Reply:
Makes sense, we should keep it as it was in the source for future porting if needed.